operator/config/network-policies/deploy.yaml

```
@@ -0,0 +1,493 @@
+      service: forklift-api
+  policyTypes:
+  - Ingress
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: forklift-consumer-policy
+  namespace: openshift-mtv
+spec:
+  egress:
+  - ports: []
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)